spec/lines_classifier_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "helper"
+require "simplecov/lines_classifier"
+
+describe SimpleCov::LinesClassifier do
+  describe "#classify" do
+    describe "only relevant lines" do
+      it "classifies each line as relevant" do
+        classified_lines = subject.classify [
+          "def foo",
+          "end",
```

Comment:
I might be over testing stuff, but a test with a class and a method definiton inside might be cool to make sure that all that works correctly :)